com.unity.render-pipelines.core/Editor/Lighting/LightAnchorEditor.cs

```diff
@@ -0,0 +1,648 @@
+                drawCapFunction(id, position, Quaternion.identity, size, type);
+
+            return position;
+        }
+    }
+    class AngleFieldState
+    {
+        public float radius;
+        public Vector2 position;
+    }
+
+    static class LightAnchorStyles
+    {
+        static public float inspectorWidthPadding = 60f;
+        static public float presetButtonCount = 9f;
+        static public GUIStyle centeredLabel = GUI.skin.GetStyle("Label");
+        static public string k_IconFolder = @"Packages/com.unity.render-pipelines.core/Editor/Lighting/Icons/";
+        static public GUIContent presetTextureRimLeft = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetRim_Left", ".png", false), "Rim Left");
+        static public GUIContent presetTextureKickLeft = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetKick_Left", ".png", false), "Kick Left");
+        static public GUIContent presetTextureBounceLeft = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetBounce_Left", ".png", false), "Bounce Left");
+        static public GUIContent presetTextureFillLeft = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetFill_Left", ".png", false), "Fill Left");
+        static public GUIContent presetTextureHair = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetHair", ".png", false), "Hair");
+        static public GUIContent presetTextureRimRight = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetRim_Right", ".png", false), "Rim Right");
+        static public GUIContent presetTextureKickRight = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetKick_Right", ".png", false), "Kick Right");
+        static public GUIContent presetTextureBounceRight = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetBounce_Right", ".png", false), "Bounce Right");
+        static public GUIContent presetTextureFillRight = new GUIContent(UnityEditor.Rendering.CoreEditorUtils.LoadIcon(LightAnchorStyles.k_IconFolder, "PresetFill_Right", ".png", false), "Fill Right");
+        static public GUIContent distanceProperty = new GUIContent("Distance", "Controls how far 'back', the light is placed from its anchor");
+        static public GUIContent upDirectionProperty = new GUIContent("Up direction", "Specifies the space in which the up direction of the anchor is defined. Local is relative to the camera.");
+        static public GUIContent[] angleSubContent = new[]
+        {
+            EditorGUIUtility.TrTextContent("Orbit"),
+            EditorGUIUtility.TrTextContent("Elevation"),
+            EditorGUIUtility.TrTextContent("Roll")
+        };
```

Comment:
instead of doing `new GUIContent`, try to use as much as possible `EditorGUIUtility.TrTextContent` when there is a string. The later will handle localization. And later someone could provide a translation pack for this in another language.

Reply:
[Fixed]